pkg/driverbuilder/buildprocessor.go

```diff
@@ -8,7 +8,7 @@ type BuildArchitecture string
 
 const BuildArchitectureX86_64 BuildArchitecture = "x86_64"
 
-var builderBaseImage = "falcosecurity/driverkit-builder:latest" // This is overwritten when using the Makefile to build
+var BuilderBaseImage = "falcosecurity/driverkit-builder:latest" // This is overwritten when using the Makefile to build
```

Comment:
If `BuilderBaseImage` changes name, do we need to reflect this change in the makefile/build process too?

Reply:
You're right, I forgot.